### PR TITLE
Azure pipelines patch

### PR DESCRIPTION
**Pull request by goagain**
This PR will fix NuGet restore doesn't work in ADO. Replaced it with `dotnet restore`.

### CODE REVIEW

**Review by Haard30 — APPROVED** [review]

File: .github/workflows/azure-devops-pipelines-cg.yml [hunk 1]
```diff
@@ -16,17 +16,10 @@ jobs:
   pool:
     name: Azure Pipelines
   steps:
-  - task: NuGetToolInstaller@1
-    displayName: Use Latest NuGet
+  - task: DotNetCoreCLI@2
+    displayName: donet restore
```
Comment (shalinikhare27) [inline .github/workflows/azure-devops-pipelines-cg.yml:20]:
```suggestion
    displayName: dotnet restore
```

File: .github/workflows/azure-devops-pipelines-cg.yml [hunk 2]
```diff
@@ -16,17 +16,10 @@ jobs:
...
-  - task: NuGetCommand@2
-    displayName: NuGet restore
-    inputs:
-      solution: '**\*.sln'
-      selectOrConfig: config
-      nugetConfigPath: nuget.config
-
+      command: 'restore'
+      projects: '**/*.sln'
+      feedsToUse: 'config'
```
Comment (shalinikhare27) [inline .github/workflows/azure-devops-pipelines-cg.yml:24]: This is not needed.

Comment (shalinikhare27) [reply .github/workflows/azure-devops-pipelines-cg.yml:24]: Only command and project will work

Comment (goagain) [reply .github/workflows/azure-devops-pipelines-cg.yml:24]: changed it to `**/*.csproj`

**Review by kyle-rader-msft — APPROVED** [review]

**Review by mvanchaa — APPROVED** [review]

**Pull request merged**